_docs/reference/notes/0.2.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Istio 0.2
```

Comment:
is this pretty much just moved from release-notes.md or there are other changes ?

might be useful to start directly with the version and just keep adding new files overtime rather than create them later